internal/util/projutil/project_util.go

```diff
@@ -351,3 +352,17 @@ func appendContent(fileContents, target, newContent string) (string, error) {
 	return fileContents[:index] + newContent + fileContents[index:], nil
 
 }
+
+// GetOperatorName returns the name of the operator which is by default the projectName attribute of the PROJECT file
+// However, the Go projects built with the plugin version v2 has not this attribute and then, for this case
+// the operatorName will be the current directory.
+func GetOperatorName(cfg *config.Config) (string, error) {
+	if cfg.ProjectName != "" {
+		return cfg.ProjectName, nil
+	}
+	dir, err := os.Getwd()
```

Comment:
```suggestion
	dir, err := os.Getwd()
	if cfg.IsV3() {
		return errors.New("project config file must contain 'projectName'")
	}
```

Reply:
@estroz the GO projects has not been scaffolded with projectName so it would need to get done in a follow up first. 
Added a todo for that.

Reply:
Out of the scope of this PR.

Reply:
Kubebuilder needs to be [upgraded](https://github.com/operator-framework/operator-sdk/pull/3530/files#diff-37aff102a57d3d7b797f152915a6dc16R46) first.